### ISSUE TITLE
cannot run the example

### ISSUE THREAD

**Issue by fwang49** [basho-labs/spark-bdp-stockticker-demo#2]
When I tried to run the example, it complained that "cannot find downloadStock.py". 

![screen shot 2017-03-01 at 12 07 26 pm](https://cloud.githubusercontent.com/assets/6878249/23819867/ca282326-05da-11e7-9cd6-687045f441b8.png)

**Comment by korry8911** [comment]
yes, downloadStocks.py was missing. I believe it got lost during a repo cleanup. I have added the file back to the repo. @fwang49
